File: 10807/helloworld.py
firstInput = input("")
first_input = int(firstInput)
count = 0
second_input = input("").split(" ")
if first_input == len(second_input):
    three_input = int(input(""))

    for checkNumber in second_input:
        if three_input == int(checkNumber):
            count += 1

    print(count)

File: 10807/helloworld.js
const fs = require('fs');
const input = fs.readFileSync("10807/test.txt").toString().split("\n");

let count = 0;

const secondInput = input[1];
const threeInput = input[2];

const loopState = secondInput.split(" ");


for(var state  in loopState){
    if(loopState[state] === threeInput){
        count++;
    }
}

console.log(count);